oauth2_client/management/commands/oauth2provider_app.py

```diff
@@ -0,0 +1,62 @@
+"""
+Create an instance of oauth2_provider's Application in DB, from CLI.
+"""
+from oauth2_client.management.commands.oauth2_app_maker import Command as ParentCommand
```

Comment:
Please change name from Command to something more specific as it can be confused with django command.

Reply:
`oauth2_app_maker` **is** a django command (it extends django's BaseCommand, via LoggingBaseCommand). Django requires every command class to be named `Command`, otherwise it won't be picked up by the framework and registered as a command. I want the `oauth2_app_maker` to be picked up by the framework (even though it is abstract) because that allows me to test it better. I tried to explain that in [this comment](http://github.com/Livit/Labster.OAuth2Client/blob/4ac9c5d9da1f41502d16fc9791f68b94f139e19b/oauth2_client/tests/test_oauth2_app_maker_cmd.py#L4-L4).

To summarize: renaming the parent command to some different name would result in (slightly!) less comprehensive tests. Please make the final call @auraz

Reply:
Sounds OK, good then!